testdata/p4_14_samples_outputs/acl1-midend.p4

```diff
@@ -134,16 +134,27 @@ header data_t {
 }
 
 struct metadata {
+    @pa_solitary("ingress", "acl_metadata.if_label") 
```

Comment:
How were tests passing before without these annotations?  The input program had a pragma or P4_14 annotation-like thing since before this PR, so did something change in this PR to make these annotations appear in the output?  Or were the tests somehow passing before even though the files did not match?

Reply:
This is probably because I rebased on top of other changes. This pr does not know anything about this annotation. I don't know where it's coming from.

Reply:
OK, I will not worry about it then.  There is a `pa_solitary` pragma in the original P4_14 program, so something in the P4_14 to P4_16 translation must be picking it up.